server/portal/apps/tickets/api/unit_test.py

```diff
@@ -14,6 +14,23 @@ def rt_tickets(scope="module"):
 def rt_ticket_history(scope="module"):
     yield json.load(open(os.path.join(settings.BASE_DIR, 'fixtures/rt/ticket_history.json')))
 
+@pytest.fixture
+def mock_unauthenticated_test(mocker):
```

Comment:
can this fixture be used in `test_tickets_create_unauthencated`.  as these are unit tests (and not something like a integration), then with that fixture there is no longer any requests made to google during the running of them.   this would resolve my concern from here https://github.com/TACC/Core-Portal/pull/530#discussion_r745268792

also, maybe the fixture name could include the word `recaptcha` to be more explicit.